pkg/tfgen/parse_markdown.go

```diff
@@ -271,7 +272,7 @@ func parseParameterSection(node *bf.Node, consumeNode func(node *bf.Node)) (para
 				return -1, nil, nil, err
 			}
 
-			if ps == nil || node == nil || node.Next == nil {
+			if ps == nil || node.Next == nil {
```

Comment:
This is safe because we check `node != nil` up above, I'm guessing?

Reply:
Yeah, exactly - it was flagged up by the linter